Sources/UIViewExtensions.swift

```diff
@@ -307,6 +309,28 @@ extension UIView {
         self.layer.transform = transform
     }
 
+    
+    /// EZSwiftExtensions
+    public func setTranslation(_ duration:TimeInterval?, _ x: CGFloat?, y: CGFloat?, completion: ((Bool) -> Void)? = nil) {
+        self.animate(duration: duration ?? UIViewTransitionDuration, animations: {
+            self.transform = .init(translationX: x ?? 0, y: y ?? 0)
+        }) { (success) in
+            if let completion = completion {
+                completion(success)
+            }
+        }
+    }
+    
+    /// EZSwiftExtensions
+    public func setTranslationIdentity(_ duration:TimeInterval?, completion: ((Bool) -> Void)? = nil) {
+        self.animate(duration: duration ?? UIViewTransitionDuration, animations: {
+            self.transform = CGAffineTransform.identity
```

Comment:
Is setTranslationIdentity a special case of setTranslation (as the only difference between the two is the definition of the transforms. 

Moreover are other kinds of transforms defined in CG? If so the name of the function needs to be changed to somehow incorporate the affine transform concept.